### DISCUSSION TITLE
Cursor Theme

### DISCUSSION THREAD

**Discussion by IgorD-lab** [prasanthrangan/hyprdots#1399] [Q&A]
I'm using a custom cursor theme, but it seems applications aren't applying it consistently. Some programs show the themed cursor, while others default to the standard cursor Additionally, the cursor theme seems to flicker between the two within the same application.

Is there a way to force all applications to use only the custom cursor theme?

https://github.com/prasanthrangan/hyprdots/assets/74680803/e51ea86b-44fb-49f5-b920-a1a965a89218

**Comment by root9191** [comment]
I am also looking for this option but i haven't found it yet.
Do you also have the problem, that your Cursor fallsback to another Version on the Desktop for example?

**Reply by IgorD-lab** [reply]
You might find looking up which cursors inside home/.icons forlder have hyprcursor folder inside (ex. Dracula-cursors) since for me those are the only ones that work. Im not really sure what is happening so it might be better to open an issue if #1517 dosent help you.

**Reply by root9191** [reply]
that worked, thanks :)

**Comment by IgorD-lab** [comment]
#1517
